packages/fx-core/templates/plugins/resource/hosting/bicep/webapp/webappProvision.template.bicep

```diff
@@ -48,7 +48,7 @@ resource webApp 'Microsoft.Web/sites@2021-02-01' = {
         }
         {
           name: 'SCM_SCRIPT_GENERATOR_ARGS'
-          value: scriptGenerator
+          value: scmScriptGeneratorArgs
```

Comment:
I prefer we use if condition here and do not introduce a varaible, which may be more readable for end user.
```
{{#if (contains "dotnet" configs)}}
        value: '--aspNetCore'
{{/if}}
{{#if (contains "node" configs)}}
        value: '--node'
{{/if}}
```